content/initiatives/workplans/codegen.md

```diff
@@ -0,0 +1,47 @@
++++
+title = "Make Rust Faster Than C"
+slug = "codegen"
+template = "initiatives/workplans/workplan.html"
+
+[extra]
+backLink = "/initiatives/codegen"
+backTitle = "Make Rust Faster Than C"
++++
+
+## Workplan
+
+### Milestone 1: Stabilize RFC & create plan   
```

Comment:
kind of a weird combination. Is it really a problem to split this up? I think it would make communicating about it much easier.